client/src/components/participant-details/cohort-table.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button } from '@material-ui/core';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+// import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+
+const useStyles = makeStyles({
+  table: {},
+});
+
+function createData(name, seats, startDate) {
+  return { name, seats, startDate };
```

Comment:
I believe this function should take endDate as a parameter also.

Reply:
This is test data method to check draft UI